api/collections.py

```diff
@@ -328,6 +328,7 @@ def get(self, cid):
         for sess in sessions:
             sess['measurements'] = session_measurements.get(sess['_id'], None)
             sess['subject_code'] = sess.get('subject', {}).get('code', '') # FIXME when subject is pulled out of session
+            sess['attachment_count'] = len([f for f in sess.get('files', []) if f.get('flavor') == 'attachment'])
```

Comment:
This has been dropped globally in master, though, which is good because the api no longer has a strict notion of attachments. It's now up to the UI to make that interpretation, using tags and metadata.